src/events.rs

```diff
@@ -16,8 +16,8 @@ impl EventData {
     pub fn description(&self) -> String {
         let description = match *self {
             EventData::AdapterStart { ref name } => name,
-            EventData::ServiceStart { ref id } => id,
-            EventData::ServiceStop { ref id } => id
+            EventData::ServiceStart { ref id }
+            | EventData::ServiceStop { ref id } => id,
```

Comment:
That's because the linter needed to be updated too. I dealt with this change in a separated commit https://github.com/JohanLorenzo/foxbox/commit/c78a39309f5a06a48af94992e73e36a3a377d7fa